test/Garnet.test/RespSortedSetTests.cs

```diff
@@ -989,12 +990,59 @@ public void CheckSortedSetOperationsOnWrongTypeObjectSE()
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetRandomMember(keys[1]));
             // ZDIFF
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetCombine(SetOperation.Difference, keys));
+            // ZDIFFSTORE
+            RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetCombineAndStore(SetOperation.Difference, destionationKey, keys));
             // ZSCAN
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetScan(keys[1], new RedisValue("*")).FirstOrDefault());
             //ZMSCORE
             RespTestsUtils.CheckCommandOnWrongTypeObjectSE(() => db.SortedSetScores(keys[1], values[1]));
         }
 
+        [Test]
+        public void CheckSortedSetDifferenceStoreSE()
```

Comment:
Please also test storing into an existing destination key

Reply:
Is it not tested already using the below line? Do you mean **non**-existing destination key?
`db.SortedSetAdd(destionationKey, key1Values); // Set up destination key to overwrite if exists`

Reply:
Added test case for those the case